update_external_versions.conf

```diff
@@ -78,12 +82,62 @@
 # in the file paths (like on arti.dev), the type parameter should be omitted entirely, otherwise
 # no images will be found.
 
-# We actually want the latest stable RPM version of csm-ssh-keys, but that function is not yet
-# available in update_external_versions. Fortunately, however, we use the same version for
-# all artifacts in CMS repos (plus or minus the build tag that is appended to unstable docker
-# and helm charts). Because of this, we will get the latest stable csm-ssh-keys docker version,
-# and just strip off the build tag if one is present, and append "-1" to the end. That is the
-# latest stable RPM version.
-image: csm-ssh-keys
+#########################################
+# Form 2: RPMs
+#########################################
+
+#rpm:  RPM_name
+#    major: major number
+#    minor: minor number
+#    outfile: target filename for version
+#    namefile: target filename for RPM filename
+#    urlfile: target filename for full URL to RPM
+#    server: algol60
+#    type: build type
+#    uri: algol60 URI
+#    url: custom URL
+#    archstrip: true or false
+#    arch: RPM arch
+#
+# For each such stanza, the only required field is the rpm field. This field
+# determines the name of the image whose latest version we wish to discover.
+#
+# The major and minor fields, if present, must contain nonnegative integers.
+# If specified, they constrain the RPM version search to versions with the
+# specified major and (if specified) minor number. If neither is specified, the
+# overall latest version of the RPM will be sought.
+#
+# If archstrip is true, then we filter for RPM filenames that end with
+# .<arch>.rpm (e.g. .x86_64.rpm, .src.rpm, .noarch.rpm, etc), and that
+# part of the filename will not be considered to be part of the version.
+# archstrip defaults to true.
+#
+# arch specifies the architecture of the RPM, for the purposes of the archstrip
+# field. If archstrip is false, this field may not be set. If archstrip is true,
+# this field defaults to the name of the directory that contains the RPM in
+# its URL path.
+#
+# outfile defines the name of the file that the version will be written to.
+# If not specified, it defaults to <RPM_name>.version
+#
+# If namefile is specified, the filename of the RPM with this version will be 
+# written to it. If unspecified, no such file is written.
+#
+# If urlfile is specified, the full URL to the RPM with this version will be
```

Comment:
So if the full URL is written to the output file, what does the version look like? Does the code follow that URL to get the version? I am confused.